.rubocop_todo.yml

```diff
@@ -0,0 +1,159 @@
+# This configuration was generated by
```

Comment:
are we planning to fix rubocop in a future PR ? also are we not pinning the ruby version for rubocop ?

Reply:
I decided not to pin the Ruby version for Rubocop since we're supporting 3 different versions, but I'm open to suggestions 😄

Reply:
3 different rubocop files (all inheriting from a common base) ?

Reply:
Also decided not to go for the rubocop fixes here since IMO they are out of scope for this PR, and they would pollute these changes and make it very hard to review.
IMO it should definitely be done in future PRs, and it shouldn't be too much work since there are many autocorrectable fixes and it's a small gem 😄 But I'll leave that up to the new maintainer team 😄

Reply:
> 3 different rubocop files (all inheriting from a common base) ?

Could be 🤔
I've just checked the[ rubocop docs on the target Ruby version](https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version) to be sure and if I understood correctly, we're supposed to specify the oldest version supported by the gem. It should also be inferred from the gemspec, but I can add it to the rubocop config to make it more explicit 😄

Reply:
done